### ISSUE TITLE
Fix accessibility issues in Home Page

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/mitxonline#155]
This ticket is a part of #150 EPIC.

The fixes that should be done under the banner of this ticket are:

- [x] Fix duplicate H1 tags on home page (there can be only one H1 per page)
- [x] Use empty ALT tags for course cards on home page by setting `alt=""`. The thumbnails are purely decorative and no alt tag would be preferable to the repetitive ones we have now.

**Comment by arslanashraf7** [comment]
Closed via #160 and #168 (Partially)
